src_c/music.c

```diff
@@ -391,16 +391,20 @@ _load_music(PyObject *obj, char *namehint)
         return NULL;
```

Comment:
I think `ext` is now leaked here as a consequence of this change.

Reply:
`ext` will always be left as NULL if the `rw` is NULL, because `ext` is only allocated in RWobject when it's returning a valid non-NULL `RWop`.